tests/test_envs.py

```diff
@@ -48,6 +48,10 @@ def test_routing(env_cls, batch_size=2, size=20):
     env = env_cls(num_loc=size)
     reward, td, actions = rollout(env, env.reset(batch_size=[batch_size]), random_policy)
     env.render(td, actions)
+    try:
+        env.improve_solution(td, actions)
```

Comment:
For now this is fine, for the future I think we can move the `local_search` part in a separate test as it is optional